lib/hanami/config.rb

```diff
@@ -288,16 +302,27 @@ def initialize(app_name:, env:)
       self.render_detailed_errors = (env == :development)
       load_from_env
 
-      @logger = Config::Logger.new(env: env, app_name: app_name)
 
       @actions = load_dependent_config("hanami-controller") {
         require_relative "config/actions"
         Actions.new
       }
 
+      @assets = load_dependent_config("hanami-assets") {
+        require_relative "config/assets"
+        Hanami::Config::Assets.new
+      }
+
+      @db = load_dependent_config("hanami-db") { DB.new }
+
+      @logger = Config::Logger.new(env: env, app_name: app_name)
+
+      @middleware = load_dependent_config("hanami-router") {
+        Slice::Routing::Middleware::Stack.new
+      }
```

Comment:
Some extra diff around here since I ordered these config objects consistently at the same time.